src/components/common/NameRequestInfo.vue

```diff
@@ -97,6 +179,14 @@ export default class NameRequestInfo extends Mixins(DateMixin) {
   @Getter getNameRequestNumber!: GetterIF;
   @Getter getNameRequestDetails!: NameRequestDetailsIF;
   @Getter getNameRequestApplicant!: NameRequestApplicantIF;
+  @Getter getNameTranslations!: Array<string>;
+
+  mounted () {
+    if (this.nameTranslations?.length) {
```

Comment:
I suggest being explicit here and checking that length > 0. (Instead of checking that length is truthy.)

Reply:
Absolutely, will update this in the places i checked against `length`

Reply:
Great. Both codes will work, but one will be clearer. (Great code is easy to review ;) )